src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -84,32 +84,7 @@ class TransportManagementService implements Serializable {
                                                 |      --output ${responseFileUpload} \\
                                                 |      '${url}/v2/files/upload'""".stripMargin()
 
-
```

Comment:
> Some questions, but in general it looks fine for me. Reviewed on plain coding level with unsufficient domain knowledge in the TMS area, maybe a colleague deeper involved in TMS should have a look ...

Hi Marcus, thanks a lot for the review! 
This change has been reviewed by my TMS colleague which can be found here: https://github.com/SAP/jenkins-library/pull/1635#commitcomment-39139482 before. More details can be found in the pull request(https://github.com/lydiazcheng/jenkins-library/pull/1) in my forked repo.

Reply:
> Makes sense to refactor getting the response body into a method since it is now used here and also by uploadMtaExtDescriptorToNode.

I refactored a getResponseBody method in Line 300. Or you mean another idea?

Reply:
All I would like to say with my comment above is that it is fine to refactor that ...